lib/article_json/export/amp/elements/embed.rb

```diff
@@ -20,20 +20,40 @@ def embed_node
             end
           end
 
-          def amp_tag
-            {
-              vimeo_video: 'amp-vimeo',
-              youtube_video: 'amp-youtube',
-            }[@element.embed_type.to_sym]
+          def type_specific_node
```

Comment:
send("#{@element.embed_type}_node".to_sym)
:)

Reply:
mmm I prefer to declare the methods, the older I'm the less I like `send` :)